concerns/supports-images.coffee

```diff
@@ -12,14 +12,12 @@ export default
 
 		# Determines whether the image should be shown via v-show
 		showImage: -> switch
+			# Image has finished loading
+			when @imageLoaded then return true
```

Comment:
Wouldn't this mean that we'd _always_ be showing the image?  Like, the `when @videoLoaded then false` would never be met with this change.

Why _does_ this fix the issue with the trapezoid shadow?  Is it because we never transition out this image, so you can never see through it to the shadow?  Maybe another way to solve would be to change the transition so that the "leave" portion of the transition has a `transition-delay` that is equal to the enter portion.  And then no `transition-duration`.  So the leaving image never fades out, it just disappears once the video finishes transitioning in.